Nodify/Connections/BaseConnection.cs

```diff
@@ -48,6 +48,32 @@ public enum ConnectionDirection
         Backward
     }
 
+    /// <summary>
+    /// The end at which the arrow head is drawn.
+    /// </summary>
+    public enum ArrowHeadEnds
+    {
+        /// <summary>
+        /// Arrow head at start.
+        /// <summary>
```

Comment:
It looks that there are a few summary tags having issues.

`XML comment has badly formed XML -- 'Expected an end tag for element 'summary'.'`